.github/workflows/ci_tests_dev.yaml

```diff
@@ -1,27 +1,25 @@
 # Test PyGMT with GMT dev version on Linux/macOS/Windows
 #
-# This workflow runs regular PyGMT tests with the GMT dev version, and also
-# pre-release versions of several dependencies like NumPy, Pandas, Xarray, etc.
-# If any tests fail, it also uploads the diff images as workflow artifacts.
-# The GMT dev version is installed by fetching the latest source codes from
-# the GMT master branch and compiling.
+# This workflow runs regular PyGMT tests with the GMT dev version, and also pre-release
+# versions of several dependencies like NumPy, Pandas, Xarray, etc. If any tests fail,
+# it also uploads the diff images as workflow artifacts. The GMT dev version is
+# installed by fetching the latest source codes from the GMT master branch and
+# compiling.
 #
-# It is triggered when a pull request is marked as "ready as review", or using
-# the slash command `/test-gmt-dev`. It is also scheduled to run on Monday,
-# Wednesday, and Friday on the main branch.
+# It is triggered when a pull request is marked as "ready as review", or labeled with
+# 'run/test-gmt-dev'. It is also scheduled to run on Monday, Wednesday, and Friday on
+# the main branch.
 #
 name: GMT Dev Tests
 
 on:
   # push:
   #   branches: [ main ]
   pull_request:
-    types: [ready_for_review]
+    types: [ labeled, synchronize ]
```

Comment:
+1 to this. We originally set "GMT Dev Tests" to run when the PR is ready for review in PR #831, but that might have been a bit of a waste of CI resources to be honest.

Reply:
I've changed the types to `types: [ opened, reopened, labeled, synchronize ]`, following  PR #2958.